### PR TITLE
first version of stack

### PR DESCRIPTION
**Pull request by cdozen**
docker-compose.dummy
- added dummy image:  ghcr.io/cdozen/dummy:main
- and run the compose in python-package.yml

### CODE REVIEW

File: .github/workflows/python-package.yml
Comment (cbernet) [inline .github/workflows/python-package.yml]: name file : 
start-stack.yml

and here :
Start the stack

File: .github/workflows/python-package.yml
Comment (cbernet) [inline .github/workflows/python-package.yml]: not needed

File: .github/workflows/python-package.yml
Comment (cbernet) [inline .github/workflows/python-package.yml]: not needed

File: .github/workflows/python-package.yml
Comment (cbernet) [inline .github/workflows/python-package.yml]: not needed

File: .github/workflows/docker-build.yml
Comment (cbernet) [inline .github/workflows/docker-build.yml]: remove this file

File: docker/tracker_dcs_stack/docker-compose.dummy.yml
Comment (cbernet) [inline docker/tracker_dcs_stack/docker-compose.dummy.yml]: use a semver tag for the image, e.g. 1.0.1

File: docker/tracker_dcs_stack/docker-compose.dummy.yml
Comment (cbernet) [inline docker/tracker_dcs_stack/docker-compose.dummy.yml]: same

File: docker/tracker_dcs_stack/docker-compose.dummy.yml
Comment (cbernet) [inline docker/tracker_dcs_stack/docker-compose.dummy.yml:13]: same

File: requirements.txt
Comment (cbernet) [inline requirements.txt]: remove all python stuff

**Review by cbernet — CHANGES_REQUESTED** [review]

**Comment by cbernet** [comment]
also please include the full stack @cdozen

File: docker/tracker_dcs_stack/influxdb/config/influx-configs
Comment (cbernet) [inline docker/tracker_dcs_stack/influxdb/config/influx-configs]: I think that this file should be removed as it will be generated locally ?

File: .github/workflows/python-package.yml
Comment (cdozen) [reply .github/workflows/python-package.yml]: Done!

File: .github/workflows/python-package.yml
Comment (cdozen) [reply .github/workflows/python-package.yml]: Done!

File: .github/workflows/python-package.yml
Comment (cdozen) [reply .github/workflows/python-package.yml]: Done!

File: .github/workflows/docker-build.yml
Comment (cdozen) [reply .github/workflows/docker-build.yml]: Done!

File: docker/tracker_dcs_stack/docker-compose.dummy.yml
Comment (cdozen) [reply docker/tracker_dcs_stack/docker-compose.dummy.yml]: Done!

File: docker/tracker_dcs_stack/docker-compose.dummy.yml
Comment (cdozen) [reply docker/tracker_dcs_stack/docker-compose.dummy.yml]: Done!

File: docker/tracker_dcs_stack/docker-compose.dummy.yml
Comment (cdozen) [reply docker/tracker_dcs_stack/docker-compose.dummy.yml:13]: Done!

File: .github/workflows/python-package.yml
Comment (cdozen) [reply .github/workflows/python-package.yml]: Done!

File: docker/tracker_dcs_stack/influxdb/config/influx-configs
Comment (cdozen) [reply docker/tracker_dcs_stack/influxdb/config/influx-configs]: done!

File: requirements.txt
Comment (cdozen) [reply requirements.txt]: done!

File: docker/tracker_dcs_stack/docker-compose.dummy.yml
Comment (cbernet) [inline docker/tracker_dcs_stack/docker-compose.dummy.yml]: use the official repo in our organization

Comment (cbernet) [reply docker/tracker_dcs_stack/docker-compose.dummy.yml]: I understand why you used yours, but please change this after we merge all PRs

File: docker/tracker_dcs_stack/docker-compose.yml
Comment (cbernet) [inline docker/tracker_dcs_stack/docker-compose.yml]: use official repo

File: docker/tracker_dcs_stack/influxdb/config/influx-configs
Comment (cbernet) [reply docker/tracker_dcs_stack/influxdb/config/influx-configs]: Was it really removed ? it doesn't seem so on github

File: requirements/base.txt
Comment (cbernet) [inline requirements/base.txt]: why do we have python requirements in this repo ?

File: requirements/docker.txt
Comment (cbernet) [inline requirements/docker.txt]: same

File: requirements/local.txt
Comment (cbernet) [inline requirements/local.txt]: same

File: setup.py
Comment (cbernet) [inline setup.py]: same

File: docker/tracker_dcs_stack/influxdb/config/influx-configs
Comment (cdozen) [reply docker/tracker_dcs_stack/influxdb/config/influx-configs]: upps! sorry I think I push it before I delete it!

File: docker/tracker_dcs_stack/docker-compose.dummy.yml
Comment (cdozen) [reply docker/tracker_dcs_stack/docker-compose.dummy.yml]: I tried to use offical repo but it gives access denied error

Comment (cbernet) [reply docker/tracker_dcs_stack/docker-compose.dummy.yml]: to be solved, then

File: docker/tracker_dcs_stack/influxdb/config/influx-configs
Comment (cdozen) [reply docker/tracker_dcs_stack/influxdb/config/influx-configs]: done!

File: docker/tracker_dcs_stack/docker-compose.yml
Comment (cdozen) [reply docker/tracker_dcs_stack/docker-compose.yml]: used: v0.0.1 from the official repo.
 I've sent you the PR for web_server because there was a "module not found" issue in the image.

File: requirements/base.txt
Comment (cdozen) [reply requirements/base.txt]: removed!

File: requirements/docker.txt
Comment (cdozen) [reply requirements/docker.txt]: done!

File: setup.py
Comment (cdozen) [reply setup.py]: done!

File: requirements/local.txt
Comment (cdozen) [reply requirements/local.txt]: done!

File: docker/tracker_dcs_stack/docker-compose.dummy.yml
Comment (cbernet) [inline docker/tracker_dcs_stack/docker-compose.dummy.yml]: use fixed tags, as I told you already, main is a moving target.

File: docker/tracker_dcs_stack/docker-compose.dummy.yml
Comment (cbernet) [inline docker/tracker_dcs_stack/docker-compose.dummy.yml]: same

File: docker/tracker_dcs_stack/docker-compose.dummy.yml
Comment (cbernet) [inline docker/tracker_dcs_stack/docker-compose.dummy.yml]: same

File: docker/tracker_dcs_stack/docker-compose.dummy.yml
Comment (cbernet) [inline docker/tracker_dcs_stack/docker-compose.dummy.yml]: same

**Review by cbernet — CHANGES_REQUESTED** [review]

File: docker/tracker_dcs_stack/docker-compose.dummy.yml
Comment (cdozen) [reply docker/tracker_dcs_stack/docker-compose.dummy.yml]: I can't push tag to the official dummy/low_voltage repo. I don't have access rights.

File: docker/tracker_dcs_stack/docker-compose.dummy.yml
Comment (cdozen) [reply docker/tracker_dcs_stack/docker-compose.dummy.yml]: Same issue - no permission

File: docker/tracker_dcs_stack/docker-compose.dummy.yml
Comment (cdozen) [reply docker/tracker_dcs_stack/docker-compose.dummy.yml]: Same issue - no permission

File: docker/tracker_dcs_stack/docker-compose.dummy.yml
Comment (cdozen) [reply docker/tracker_dcs_stack/docker-compose.dummy.yml]: Same issue - no permission

**Comment by cbernet** [comment]
@cdozen to create the tag, you need to create a github release as I've shown you. you should not push directly to the repo. I've added you as maintainer to the dummy and low_voltage repos, so you should be able to make a release. Let me know. 

No docker push from your machine ! :-)

**Comment by cbernet** [comment]
@cdozen 

image

Then choose semver tag : 
image

File: docker/tracker_dcs_stack/docker-compose.dummy.yml
Comment (cdozen) [reply docker/tracker_dcs_stack/docker-compose.dummy.yml]: added official repo and fix tag: ghcr.io/cms-tedd-ip2i/dummy:0.0.1

File: docker/tracker_dcs_stack/docker-compose.dummy.yml
Comment (cdozen) [reply docker/tracker_dcs_stack/docker-compose.dummy.yml]: usedd fix tag: ghcr.io/cms-tedd-ip2i/dummy:0.0.1

File: docker/tracker_dcs_stack/docker-compose.dummy.yml
Comment (cdozen) [reply docker/tracker_dcs_stack/docker-compose.dummy.yml]: usedd fix tag: ghcr.io/cms-tedd-ip2i/dummy:0.0.1

File: docker/tracker_dcs_stack/docker-compose.dummy.yml
Comment (cdozen) [reply docker/tracker_dcs_stack/docker-compose.dummy.yml]: usedd fix tag: ghcr.io/cms-tedd-ip2i/low_voltage:0.0.1

**Pull request merged**
